forge-gui/res/cardsfolder/upcoming/bank_job.txt

```diff
@@ -0,0 +1,13 @@
+Name:Bank Job
+ManaCost:1 B R
+Types:Enchantment
+T:Mode$ Phase | Phase$ Upkeep | ValidPlayer$ You | Execute$ TrigExile | TriggerZones$ Battlefield | TriggerDescription$ At the beginning of your upkeep, exile the bottom creature card of your library. You may cast that card this turn. At the beginning of the next end step, if that card is still exiled, put it into your graveyard and create a Treasure token.
+SVar:TrigExile:DB$ ChangeZoneAll | ChangeType$ Card.YouOwn+BottomLibrary_Creature | Origin$ Library | Destination$ Exile | RememberChanged$ True | SubAbility$ DBEffect
```

Comment:
Weird wording

But shouldn't simply `Card.YouOwn+BottomLibrary+Creature` do the *job*? :P

Reply:
I think this only works if the bottom card of your library happens to be a Creature as well

Reply:
Test with non-creature card on the bottom of the library... nothing gets exiled